packages/theming/demo/stories/GetColorStory.tsx

```diff
@@ -57,19 +65,63 @@ const Color = ({ dark, hue, light, offset, shade, theme, transparency, variable
         {backgroundColor}
       </Tag>
     );
+
+    const hues = [...Object.keys(theme.colors), ...Object.keys(theme.palette)].filter(
+      _hue => _hue !== 'base' && _hue !== 'variables'
+    );
+    const selectedHue = (theme.colors.base === 'dark' ? dark?.hue : light?.hue) || hue || '';
+
+    if (!(variable || hues.includes(selectedHue))) {
+      generatedHue = [...Array(12).keys()].reduce<Record<number, string>>((retVal, index) => {
+        const _shade = (index + 1) * 100;
+
+        retVal[_shade] = getColor({ theme, hue: opacify(backgroundColor, 1), shade: _shade });
+
+        return retVal;
+      }, {});
+
+      /* eslint-disable-next-line no-console */
+      console.log(generatedHue);
+    }
   } catch (error) {
     background = 'transparent';
     tag = (
-      <Tag hue="red" size="large">
+      <Tag hue="dangerHue" size="large">
         {error instanceof Error ? error.message : String(error)}
       </Tag>
     );
   }
 
-  return <StyledDiv background={background}>{tag}</StyledDiv>;
+  return (
+    <>
+      <StyledDiv $background={background}>{tag}</StyledDiv>
+      {!!generatedHue && (
+        <>
+          <LG style={{ margin: '20px 0 12px' }}>Generated hue</LG>
```

Comment:
[nit] Should we align our vocabulary with the Color Principles [docs](https://garden.zendesk.com/design/color#palette)?

```suggestion
          <LG style={{ margin: '20px 0 12px' }}>Generated palette</LG>
```
Alternatively:

```suggestion
          <LG style={{ margin: '20px 0 12px' }}>Generated color scale</LG>
```

Reply:
I'd prefer to align with the technical [theme object](https://garden.zendesk.com/components/theme-object#hues) and [palette](https://garden.zendesk.com/components/palette) documentation. Btw, the term is also used in [color principles](https://garden.zendesk.com/design/color#overview).

Reply:
Oh I see. 👍